src/collections.js

```diff
@@ -0,0 +1,86 @@
+import {
+  actionToPromise,
+} from 'core';
+
+/** Create an action that runs several actions at once and resolves as soon
+ * as any one of the actions resolves.
+ *
+ * NOTE: once its actions have started, this action is not responsible for
+ * stopping them. For example, if you've started playing several audio
+ * files at once and you want them all to stop when the first one stops,
+ * this action doesn't provide for that. Caveat programmator!
+ *
+ * @param {action} action An action.
+ *                        The #any action accepts an arbitrary number of
+ *                        action arguments.
+ *
+ * @return {action}      An action.
```

Comment:
The extra space might confuse the automatic documentation generators. 
```suggestion
 * @return {action} An action
```